### ISSUE TITLE
Menu manager plugin permissions do nothing

### ISSUE THREAD

**Issue by fdlk** [molgenis/molgenis#8072]
### How to Reproduce
Give non-system user plugin permission on the menu manager. Log in as that user, update the menu and save changes.

### Expected behavior
It works.

### Observed behavior
> Error! Access is denied

**Comment by dennishendriksen** [comment]
```
@Override
@PreAuthorize("hasAnyRole('ROLE_SYSTEM, ROLE_SU, ROLE_PLUGIN_WRITE_menumanager')")
@Transactional
public void saveMenu(Menu molgenisMenu)
```
Does the non-system user have role ```ROLE_PLUGIN_WRITE_menumanager```?

**Comment by dennishendriksen** [comment]
Fixed by 9043b2902850a03b24b0b9faf6e9f591d22e4019
